### DISCUSSION TITLE
[Bootstrap 5] Where does the .btn-secondary come from (SCSS)?

### DISCUSSION THREAD

**Discussion by strarsis** [twbs/bootstrap#33036] [General] [answered]
In the Bootstrap v5 CSS is a CSS `Ruleset` `.btn-secondary { [...] }`. It can't be found in the `SCSS` files however.
How is `.btn-secondary` generated? Is it exclusive to the CSS file or by some obfuscated SCSS function call?

Related issue?: https://github.com/twbs/bootstrap/discussions/31159

**Comment by XhmikosR** [comment, marked as answer]
https://github.com/twbs/bootstrap/blob/e50c11b8c6434b6d68ea5897771e4d35fe12f5c3/scss/_variables.scss#L81-L90

and then 

https://github.com/twbs/bootstrap/blob/e50c11b8c6434b6d68ea5897771e4d35fe12f5c3/scss/_buttons.scss#L58-L62

**Comment by strarsis** [comment]
@XhmikosR: Thanks 👍
